code/game/objects/items/devices/paicard.dm

```diff
@@ -128,7 +128,7 @@
 			if(10) src.overlays += "pai-null"
 
 /obj/item/device/paicard/proc/alertUpdate()
-	visible_message("<span class ='info'>[src] flashes a message across its screen, \"Additional personalities available for download.\"", 3, "<span class='notice'>[src] bleeps electronically.</span>", 2)
+	audible_message("<span class ='info'>[src] flashes a message across its screen, \"Additional personalities available for download.\"", 3, "<span class='notice'>[src] bleeps electronically.</span>", 2)

```

Comment:
@phil235 the bug is the message is not actually working anymore the proc activates but the message no longer works something broke this elsewhere.